profiler/src/ProfilerEngine/Datadog.Profiler.Native/ApplicationInfo.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+#include <string>
+
+class ApplicationInfo
```

Comment:
These 3 values are used for "Unified Service Tagging"
https://docs.datadoghq.com/getting_started/tagging/unified_service_tagging/

Not sure if what something you want to use here instead of the very generic `ApplicationInfo`.